contracts/src/v0.8/keystone/CapabilityRegistry.sol

```diff
@@ -329,24 +326,23 @@ contract CapabilityRegistry is OwnerIsCreator, TypeAndVersionInterface {
 
   /// @notice Removes a node operator
   /// @param nodeOperatorIds The IDs of the node operators to remove
-  function removeNodeOperators(uint256[] calldata nodeOperatorIds) external onlyOwner {
-    for (uint256 i; i < nodeOperatorIds.length; ++i) {
-      uint256 nodeOperatorId = nodeOperatorIds[i];
+  function removeNodeOperators(uint32[] calldata nodeOperatorIds) external onlyOwner {
```

Comment:
Do we not want to handle the validity of the `nodeOperatorId` and return error to inform the callers? Specifically,
- When removing a `nodeOperatorId` that is out of range of `s_nodeOperatorId` (non-existing nodeOp)
- When removing a `nodeOperatorId` that is already removed before -> this will result in multiple emits of `NodeOperatorRemoved` event.

Reply:
Same with removeDONs().

Let's clarify the expected behavior. If no logic change is needed, we should update the function header doc to avoid confusion.

Reply:
Those are out of scope of this PR and will be done in a separate PR